templates/default/sv-mss-log-run.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Probably? But the pattern we usually use for runit scripts is that they're here, whether they need to be or not, lol. Will consider moving it.